application/features/VNC.py

```diff
@@ -108,16 +109,20 @@ def reset_vnc_password(self, password):
             "rm -rf ~/.vnc",
             "mkdir ~/.vnc",
 
-            f"printf '{XSTARTUP_STR}' > ~/.vnc/xstartup",
-            "cp /etc/vnc/xstartup ~/.vnc  >& /dev/null",
-            "chmod 700 ~/.vnc/xstartup",
+            # FIXME: re-enable xstartup config
+            # f"printf '{XSTARTUP_STR}' > ~/.vnc/xstartup",
+            # "cp /etc/vnc/xstartup ~/.vnc  >& /dev/null",
+            # "chmod 700 ~/.vnc/xstartup",
 
-            "echo '%s'| xxd -r -p > ~/.vnc/passwd" % hexed_passwd,
+            "echo '%s'| vncpasswd -f > ~/.vnc/passwd" % password,
             "chmod 600 ~/.vnc/passwd",
         ]
+        logger.debug(f"reset_cmd_lst={reset_cmd_lst}")
```

Comment:
Sorry for missing this in the last review. We should avoid logging anything that contains passphrases. 
```suggestion
```